onadata/apps/logger/migrations/0014_populate_attachment_xform.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.2.11 on 2024-04-22 06:42
+
+from django.db import migrations
+
+
+def populate_attachment_xform(apps, schema_editor):
+    """Populate xform field for Attachments"""
+    Attachment = apps.get_model("logger", "Attachments")
+    queryset = Attachment.objects.filter(xform__isnull=True)
+    count = queryset.count()
+    print("Start populating attachment xform...")
+    print(f"Found {count} records")
+
+    for attachment in queryset.iterator(chunk_size=100):
+        # We do not want to trigger Model.save or any signal
+        # Queryset.update is a workaround to achieve this.
+        # Instance.save and the post/pre signals may contain
+        # some side-effects which we are not interested in e.g
+        # updating date_modified which we do not want
+        Attachment.objects.filter(pk=attachment.pk).update(
+            xform=attachment.instance.xform
```

Comment:
Can we use `xform = attachment.instance.xform_id` instead? 

Also, can we add the submission instance user i.e. `user=attachment.instance.user_id`? It is the other new field introduced in this PR.